src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
@conan-z Googling, I confirmed that almost all plugins which implements the extended ParameterValue class has `value.jelly` only in short class name directory. (Of course including jenkins-maven-metadata-plugin and nodelabelparameter-plugin too.)

So, my proposal for this issue is:
* Revert this patch (No change for GT)
* Create new PR for rebuild plugin that `Base64EncodedStringParameterValue.jelly` is added to the resource of RebuildAction.